### ISSUE TITLE
Issue with imports and definitions

### ISSUE THREAD

**Issue by danjohnso** [syncfusion/ej2-buttons#2]
When you removed the @import "definition.scss"; in the 16.3.25 release, you broke several other plugin references to this package as their SASS imports do not pull the definition.scss file in leading to errors when trying to compile.

**Comment by ChristoIssac** [comment]
Hi @danjohnso,
We have checked the reported issue and we found some compilations issues in ej2-buttons source. Could you please confirm whether you are facing the same issue in below screenshot? 
 
![image](https://user-images.githubusercontent.com/29474815/47997439-d5169f00-e121-11e8-8c31-5da8ac214291.png)

If not, please provide more details about the compilation issue that you are facing or provide the screenshot if possible? It would be helpful for us to resolve this issue at earliest.

Thanks,
Christo

**Comment by danjohnso** [comment]
Basically, all of your sub packages for specific frameworks are wrong.  Vue, Angular, React, they all reference the SCSS in your "base packages" (buttons, dropdowns, grids etc) incorrectly (or at the very less inconsistently.  I found the best way to resolve for now is to not reference the SCSS in the sub-package directly, but to just import from the base like so.

![image](https://user-images.githubusercontent.com/852995/48004743-6b71a300-e0d7-11e8-9674-28c24b760c3b.png)

If I wanted the specific reference to compile, I would have to build my SCSS file like this:
![image](https://user-images.githubusercontent.com/852995/48004970-dcb15600-e0d7-11e8-8ec7-94c5693047f8.png)

This gets to be painful tracking down references since I had issues with several plugins doing the same thing.  Before I changed to reference the base packages, I had it tracked down to that _definition file I in the screenshot above as the missing link, but I had problems in 5 other plugins after that got resolved.

Your build might work within this repository, but all of your other repositories point back to this SCSS differently from their documentation so it was a very frustrating 3 hours trying to figure out where sass variables were missing from.
